baseplate/metrics.py

```diff
@@ -95,7 +95,11 @@ def flush(self):
         # TODO: run-length compression
         metrics, self.buffer = self.buffer, []
         message = b"\n".join(metrics)
-        self.transport.send(message)
+        try:
+            self.transport.send(message)
+        except socket.error as e:
+            logger.error("BufferedTransport flush exception, length %d, %s",
```

Comment:
This log message would probably be pretty confusing for someone who isn't in this code seeing it in their service's logs. Something more actionable for them would be helpful. Maybe like `baseplate metrics batch too large: flush more often or reduce amount done in one request"?